optimade/validator/validator.py

```diff
@@ -472,16 +551,16 @@ def test_as_type(self):
             self.deserialize_response(response, self.as_type_cls)
 
     @test_case
-    def test_page_limit(self, response, check_next_link=True):
+    def test_page_limit(self, response, check_next_link=5):
```

Comment:
I'll just start getting these in here 😅 
```suggestion
    def test_page_limit(self, response, check_next_link: int = 5):
```